controllers/processors/deltaProcessor.go

```diff
@@ -0,0 +1,67 @@
+package processors
```

Comment:
I like this code and it is obvious to me but if you add some description on this file, it would be much easier to read the code.

Reply:
done.